packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -308,7 +308,7 @@ watch(
 	{ immediate: true }
 );
 
-const formattedAbstract = computed<string>(() => document.value?.description ?? '');
```

Comment:
We still have the `description` value on document. What was being put here before?